Tasks/VsTest/vstest.ts

```diff
@@ -172,6 +183,15 @@ function getVstestArguments(settingsFile: string, tiaEnabled: boolean): string[]
     return argsArray;
 }
 
+function isDebugEnabled(): boolean {
```

Comment:
Fair feedback.. This is existing code (related to TIA) will check with Siddhartha.